### DISCUSSION TITLE
Default Login | Ubuntu LXC

### DISCUSSION THREAD

**Discussion by homelab-id10t** [tteck/Proxmox#1346] [Help Needed]
I looked at the README and the actual installer script but nowhere did it mention the default login details for this container. 

I couldn't find anybody else asking so I'm either really dumb or this isn't as obvious as it should be. Mind helping an idiot out? 

I've tried: ubuntu/ubuntu, root/ubuntu. root/administrator, root/root

I used Ubuntu LXC
Version 22.04
To create a new Proxmox VE Ubuntu LXC, run the command below in the Proxmox VE Shell.

bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/ubuntu.sh)"

**Comment by tteck** [comment]
If you're being prompted for credentials that you didn't input, it means you made an error. Delete the LXC and begin again, making sure to remain on the shell until the script completes.
